app/assets/stylesheets/views/_covid.scss

```diff
@@ -45,7 +45,13 @@ $c19-landing-page-header-background: #0843a1;
   background: $c19-landing-page-header-background;
 
   @include govuk-media-query($from: tablet) {
-    padding: 45px 0 40px 0;
+    padding: 45px 0 40px;
+  }
+
+  .govuk-link {
```

Comment:
is it possible to use another class like `.covid__page-header-action-link`?

Reply:
updated to `,gem-c-action-link__link` as per our convo.